devolutions-agent/src/updater/mod.rs

```diff
@@ -240,10 +243,38 @@ async fn check_for_updates(product: Product, update_json: &UpdateJson) -> anyhow
             }
         }
         VersionSpecification::Specific(version) => {
-            if version != remote_version {
-                warn!(%product, %version, "Product uptate target version does not match available version on devolutions.net, skipping update");
+            if version == detected_version {
+                info!(%product, %detected_version, "Product is up to date, skipping update");
                 return Ok(None);
             }
```

Comment:
suggestion: For `VersionSpecification::Specfic`, this logic is duplicated, and can be removed.

Reply:
Oh, I’m referring to this: https://github.com/Devolutions/devolutions-gateway/pull/1116#discussion_r1852319041